pkgs/applications/radio/soapysdrplay/default.nix

```
@@ -0,0 +1,31 @@
+{ stdenv, lib, fetchFromGitHub, cmake, soapysdr, sdrplay }:
+
+stdenv.mkDerivation rec {
+  name = "soapysdr-sdrplay3";
+  version = "0.0.1";
+
+  src = fetchFromGitHub {
+    owner = "pothosware";
+    repo = "SoapySDRPlay3";
+    rev = "e6fdb719b611b1dfb7f26c56a4df1e241bd10129";
+    sha256 = "0rrylp3ikrva227hjy60v4n6d6yvdavjsad9kszw9s948mwiashi";
+  };
+
+  nativeBuildInputs = [ cmake ];
+
+  buildInputs = [ soapysdr sdrplay ];
+
+  cmakeFlags = [
+    "-DLIBSDRPLAY_INCLUDE_DIRS=${sdrplay}/include"
+    "-DLIBSDRPLAY_LIBRARIES=${sdrplay}/lib/${sdrplay.soName}"
+    "-DSoapySDR_DIR=${soapysdr}/share/cmake/SoapySDR/"
+  ];
+
+  meta = with lib; {
+    description = "Soapy SDR module for SDRPlay";
+    homepage = "https://github.com/pothosware/SoapySDRPlay3";
+    license = licenses.mit;
+    maintainers = [ pmenke ];
```

```suggestion
    maintainers = [ maintainers.pmenke ];
```
I think this causes the ofBorg check to fail.

---

Ok. I updated it according to your suggestion.